### ISSUE TITLE
Why resalloc docker run resalloc-server as root?

### ISSUE THREAD

**Issue by pkking** [fedora-copr/copr#2593]
Any background i missed?
Seems the logrotate and cronjob run as user resalloc, but the docker start resalloc-server as root.

**Comment by praiskup** [comment]
Doing this rootless typically costs some efforts :-) and nobody did this (in OpenShift,
you never have root but some randomly generated user ID).  Patches are welcome.

**Comment by pkking** [comment]
So in fedora production Copr environment, resalloc-server start as normal user, right?

**Comment by praiskup** [comment]
Yes, on our production _virtual machine_, we run resalloc server as the `resalloc` system user.
